easybuild/easyblocks/o/openblas.py

```diff
@@ -13,8 +14,31 @@ class EB_OpenBLAS(ConfigureMake):
     def build_step(self):
         """Custom build procedure for OpenBLAS."""
 
+        self.cfg['buildopts'] += 'BINARY=64 USE_THREAD=1 USE_OPENMP=1 CC="%s" FC="%s"' % (
+            os.environ['CC'], os.environ['FC'])
+
         if get_cpu_architecture() == POWER:
             # There doesn't seem to be a POWER9 option yet, but POWER8 should work.
             self.cfg['buildopts'] += ' TARGET=POWER8'
 
         super(EB_OpenBLAS, self).build_step()
+
+    def install_step(self):
+        """Custom install procedure for OpenBLAS."""
+        self.cfg['installopts'] += 'USE_THREAD=1 USE_OPENMP=1 PREFIX=%s' % self.installdir
```

Comment:
Also here, you'll need to use `self.cfg.update`.

And I think it makes sense to pick up `buildopts` here, and just add `PREFIX=...` to it in `installopts`, to ensure consistency?